deployment/cdk/opensearch-service-migration/lib/migration-services-yaml.ts

```diff
@@ -61,7 +61,7 @@ export class OSIBackfillYaml {
 
 export class ECSReplayerYaml {
     ecs: ECSService;
-    scale: number = 1;
```

Comment:
^ this is the replayer. We definitely don't want 5 replayers.

I was linking to this line to show what the RFS version should look like.

Reply:
I'd also somewhat rather the value is explicitly set in the RFS stack than in here -- if someone was looking to change it, that's where (I think) they'd look